src/sql/SqlService.ts

```diff
@@ -255,7 +255,7 @@ export class SqlServiceImpl implements SqlService {
      * @param connection
      * @returns {@link HazelcastSqlException}
      */
-    toHazelcastSqlException(err: any, connection: Connection) : HazelcastSqlException {
+    rethrow(err: any, connection: Connection): HazelcastSqlException {
```

Comment:
refactored so that rethrow takes err and connection and returns a HazelcastSqlException whereas toHazelcastSqlException is just for converting an err to HazelcastSqlException